text/testing/mock.md

```diff
@@ -36,27 +36,56 @@ impl Trait for TestRuntime {
 Next, we create a new type that wraps the mock `TestRuntime` in the pallet's `Module`.
 
 ```rust, ignore
-pub type HelloSubstrate = Module<TestRuntime>;
+pub type TestPallet = Module<TestRuntime>;
 ```
 
 It may be helpful to read this as type aliasing our configured mock runtime to work with the pallet's `Module`, which is what is ultimately being tested.
 
 ## `impl system::Trait`
 
-In many cases, the pallet's `Trait` inherits `system::Trait` like
+In many cases, the pallet's `Trait` inherits `system::Trait` like:
 
 ```rust, ignore
 pub trait Trait: system::Trait {
 	type Event: From<Event<Self>> + Into<<Self as system::Trait>::Event>;
 }
 ```
 
-The mock runtime must inherit and define the `system::Trait` associated types (*[remember](https://substrate.dev/recipes/traits/index.html)*). To do so, `impl` the `system::Trait` for `TestRuntime` with a few types imported from other crates,
+Before we can create a mock runtime that take our pallet to run tests, we first need to create the
+outer environment for the runtime, as follows:
```

Comment:
The information you've added here is good and should be kept. But it is added right in the middle of another thought. You should finish showing how to `impl system::Trait` then explain how to impl the outer enums.

Reply:
// First implement the system pallet trait for test runtime